### DISCUSSION TITLE
Change Label text?

### DISCUSSION THREAD

**Discussion by erflot** [emilk/egui#211] [Q&A] [answered]
Hi,
Label background_color and text_color can be changed, but not text. 
Changing text could be useful for feedback to the user w/o popup.
Are there other options? 
Thanks for any hints.

**Comment by emilk** [comment, marked as answer]
You are not expected to store a `Label`, but construct a new each frame. See https://docs.rs/egui/0.10.0/egui/#understanding-immediate-mode

**Reply by erflot** [reply]
Thanks, i normally do it this way.
Was stuck with a label to be changed across panels. I will use a variable instead like ui.label(var).
